### PR TITLE
[ZEPPELIN-1020] Remove suggestion list on click

### PR DESCRIPTION
**Pull request by suvam97**
### What is this PR for?
The suggestion list under note permissions for readers, writers & owners should hide on-click anywhere on the notebook.

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
[ZEPPELIN-1020](https://issues.apache.org/jira/browse/ZEPPELIN-1020)

### How should this be tested?
1.Click on any field for setting permission for owners, readers & writers.
2.The suggestion list is displayed.
3.Now, click on any part of the notebook the suggestion list hides.

### Screenshots (if appropriate)

**BEFORE:**
http://g.recordit.co/7Esaq245Tp.gif

**AFTER**
http://g.recordit.co/Rfgn5dONPz.gif

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by prabhjyotsingh** [comment]
LGTM!

**Comment by r-kamath** [comment]
LGTM

**Comment by suvam97** [comment]
@prabhjyotsingh @r-kamath Thanks for the review!
@prabhjyotsingh done required changes.

**Comment by ravicodder** [comment]
@suvam97 Thanks for the contribution!! LGTM

**Comment by prabhjyotsingh** [comment]
Merging this if no more discussion.

**Comment by corneadoug** [comment]
I still see two line with '$'

File: zeppelin-web/src/app/notebook/notebook.controller.js [hunk 1]
```diff
@@ -879,6 +879,7 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl',
 
   // function to find suggestion list on change
   $scope.search = function(role) {
+    $('.userlist').show();
```
Comment (corneadoug) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: angular.element?

File: zeppelin-web/src/app/notebook/notebook.controller.js [hunk 2]
```diff
@@ -950,5 +951,8 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl',
     $scope.suggestions = [];
   };
 
+angular.element(document).click(function(){
+     $('.userlist').hide();
```
Comment (ravicodder) [inline zeppelin-web/src/app/notebook/notebook.controller.js]: angular.element?

**Comment by suvam97** [comment]
@corneadoug @ravicodder Thanks for reviewing ! Somehow missed to make changes .. Final changes updated ..

**Comment by corneadoug** [comment]
Merging if there is no more discussions

**Pull request closed without merging**